lib/Target/RISCV/RISCVInstrFormats.td

```diff
@@ -65,30 +59,11 @@ class InstR<string mnemonic, bits<7> op, bits<10> funct10,
   bits<5> RS1;
   bits<5> RS2;
 
-  let Inst{31-27} = RD;
-  let Inst{26-22} = RS1;
-  let Inst{21-17} = RS2;
-  let Inst{16- 7} = funct10;
-  let Inst{6 - 0} = op;
-}
-
-//R-Type
-class InstR64<string mnemonic, bits<7> op, bits<10> funct10, 
-            SDPatternOperator operator, RegisterOperand cls1, 
-            RegisterOperand cls2>
-  : InstRISCV<4, (outs cls1:$dst), (ins cls2:$src2, cls2:$src1), 
-                mnemonic#"\t$dst, $src1, $src2", 
-                [(set cls1:$dst, (sext (operator cls2:$src1, cls2:$src2)))]> {
-  field bits<32> Inst;
-
-  bits<5> RD;
-  bits<5> RS1;
-  bits<5> RS2;
-
-  let Inst{31-27} = RD;
-  let Inst{26-22} = RS1;
-  let Inst{21-17} = RS2;
-  let Inst{16- 7} = funct10;
+  let Inst{31-25} = funct7;
+  let Inst{24-20} = RS1;
```

Comment:
RS2 is in the upper bit position according to the ISA spec, which also matches the other instruction formats.